.jenkins/common_job_properties.groovy

```diff
@@ -81,13 +81,15 @@ class common_job_properties {
 
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
+                                        def branchWhitelist,
                                         def commitStatusContext,
                                         def successComment = '--none--') {
     context.triggers {
       githubPullRequest {
         admins(['asfbot'])
         useGitHubHooks()
         orgWhitelist(['apache'])
+        whiteListTargetBranches(branchWhitelist)
```

Comment:
This should have a default. Optimally, this default would be defined outside of this method, branchWhitelist would be set to it in the method definition, and the default would be commented. Please investigate doing so.